test/kind/kind.yaml

```diff
@@ -0,0 +1,13 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+    image: kindest/node:v1.19.1
+  - role: worker
+    image: kindest/node:v1.19.1
+  - role: worker
+    image: kindest/node:v1.19.1
+networking:
+  ipFamily: ipv4
+  podSubnet: "10.10.0.0/16"
+  serviceSubnet: "10.11.0.0/16"
```

Comment:
Curious - Reasons to use specific subnet for pod and service?

Reply:
It's just to have it explicitly for consistency